### ISSUE TITLE
For classification task fluid version trains slower than v2 version with same learning rate.

### ISSUE THREAD

**Issue by TheodoreG** [PaddlePaddle/Paddle#14830]
We found that for the same input, same network structure, same Momentum optimizer and all the sam e hyper parameters,  fluid classification version has 1% ~ 2% lower accuracy compared to v2 version and trains slower. When we set learning rate 10x larger for the fluid version, the loss descendant curve resembles v2's curve more. 
So is there any explanation for my observation?

**Comment by typhoonzero** [comment]
any details about what model you are training, whether GPU is used, and training environment configurations, so that we can reproduce the case.

In most case, the latest fluid version should be faster than v2 version, and for model evaluations, we may need to check out if there are implementation differences for your case, some models may have some detailed places to update compared to v2.
